### ISSUE TITLE
Make yarn version information with each command a debug-level message

### ISSUE THREAD

**Issue by BYK** [yarnpkg/yarn#4065]
Currently, we output the current version of yarn via a line like `yarn 0.28.0`. This is unnecessary noise, especially on CI runs unless you are debugging. Even when you are debugging you can get this easily via `yarn --version` so this line should be completely removed or reduced to a debug-level message.

**Comment by Daniel15** [comment]
> This is unnecessary noise

I disagree. It allows us to easily see the version number in bug reports or in other people's CI builds, without having to explicitly ask for the version. I can look at the output of https://circleci.com/gh/yarnpkg/yarn/4681 and immediately know that the CircleCI build used Yarn v0.27.5.

**Comment by BYK** [comment]
>I disagree. It allows us to easily see the version number in bug reports or in other people's CI builds, without having to explicitly ask for the version. I can look at the output of https://circleci.com/gh/yarnpkg/yarn/4681 and immediately know that the CircleCI build used Yarn v0.27.5.

Well, the plan is to automatically switch/output all debug information whenever we see a crash. In all other cases, the version is not really important I think, it's just extra noise that people don't care.

We can always add `yarn --version` to CircleCI builds if we need it at all times but I think revealing the relevant information in the right context is better than simply printing it out all the time. What do you think?
